### PR TITLE
[TwigBridge] Symfony 3.1 forward compatibility

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | https://github.com/symfony/symfony/pull/17743#discussion_r52718344 |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

File: src/Symfony/Bridge/Twig/Extension/YamlExtension.php [hunk 1]
```diff
@@ -40,7 +40,13 @@ public function encode($input, $inline = 0, $dumpObjects = false)
         }
 
         if (defined('Symfony\Component\Yaml\Yaml::DUMP_OBJECT')) {
-            $dumpObjects = (int) $dumpObjects;
+            if (is_bool($dumpObjects)) {
```
Comment (javiereguiluz) [inline src/Symfony/Bridge/Twig/Extension/YamlExtension.php]: Maybe the ternary operator would be more readable in this case?

```php
$flags = is_bool($dumpObjects) ? Yaml::DUMP_OBJECT : 0;
```

Comment (fabpot) [inline src/Symfony/Bridge/Twig/Extension/YamlExtension.php]: agreed

Comment (xabbuh) [inline src/Symfony/Bridge/Twig/Extension/YamlExtension.php]: done

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**